tools/debug_tool/debug_server.py

```diff
@@ -0,0 +1,70 @@
+from flask import Flask, render_template, request, send_file, abort
+from flask_cors import CORS
+from requests import get
+import os
+
+app = Flask(__name__)
+app.config["CORS_ORIGINS"] = ["*"]
+CORS(app, support_credentials=True)
+
+KOTEKAN_ADDRESS = "http://localhost:12048"
+
+# Display kotekan pipeline
+@app.route("/", defaults={"path": ""})
+@app.route("/debug_tool/pipeline_tree.html", methods=["GET", "POST"])
```

Comment:
Can we just serve this file via the `dir_listing()` function below?   I'd like this webserver to be fairly generic so we can just add more html/css/javascript files to a directory and not have to add custom functions to this pipeline.